### PR TITLE
Remove map, simplify.

### PR DESCRIPTION
**Pull request by 8uurg**
< layout, interface

### CODE REVIEW

**Comment by Tarovk** [comment]
Checkstye still has some issues.

File: src/main/java/website/frontrow/ui/status/StatusPanel.java [hunk 1]
```diff
@@ -26,61 +23,16 @@ public StatusPanel(ArrayList<Player> players)
...
-    }
-
-    @Override
-    public void notInvincible(Player player)
-    {
-        //Nothing yet
-    }
 
     /**
      * Updates all the components!
```
Comment (LarsStegman) [inline src/main/java/website/frontrow/ui/status/StatusPanel.java]: This method can be removed.

File: src/main/java/website/frontrow/ui/status/PlayerStatusPanel.java [hunk 2]
```diff
@@ -64,6 +70,7 @@ public void setScore(int score)
     {
         this.score = score;
         update();
+        revalidate();
```
Comment (LarsStegman) [inline src/main/java/website/frontrow/ui/status/PlayerStatusPanel.java]: Move to update please.

< layout

**Comment by 8uurg** [comment]
The LevelBuilder errors have been fixed in another branch.

**Pull request merged**